### ISSUE TITLE
@analytics/google-analytics Custom Dimensions not working for node.js

### ISSUE THREAD

**Issue by Bug-Reaper** [DavidWells/analytics#86]
Love this library! Was incorporating it on the Node side of things instead of Browser and couldn't see any of my custom dimensions being reported to GA.

Got curious and eventually found this:
https://github.com/DavidWells/analytics/blob/29a3b28bb1e7c111edb2afdd2f5e58c6329fbc3e/packages/analytics-plugin-google-analytics/src/node.js#L63

Which provided some useful insight: ```   // Todo map properties to custom dimensions```

Cool if I do a PR for this? I'm assuming this can be lifted pretty much straight the way the browser processes custom dimensions. Thanks again for the cool library :)

**Comment by Bug-Reaper** [comment]
What I can tell, is that the browser version uses google's vanilla lib and the server version uses the universal-analytics package. It's a little unclear if universal-analytics supports custom dimensions but it looks to be true. 

~~A little weird testing because NPM libs don't match what's in this repo and there's multiple versions? Could use some insight into how would be the best way to do a NPM install of a fork for in a situation like this~~

I tested this by running `npm build` on the plugin and overwriting a true install of the @analytics/google-analytics plugin with the output. I'm sure there's a better way to do this, but it works.

**Comment by Bug-Reaper** [comment]
Confirmed universal-analytics supports custom dimensions.... or at least it did in 2018. Documentation is bad. https://github.com/peaksandpies/universal-analytics/pull/82

**Comment by Bug-Reaper** [comment]
Got this working, the gist is:
```
  const dimensions = formatObjectIntoDimensions(properties, customDimensions);
  console.log(dimensions)
  client.event(category, event, label, value, dimensions).send();
/*
dimensions =
{
  cd1  :  "the value of your first custom dimension"
  cd2  :  "the value of your second custom dimension"
 ...
}
*/
```
Instead of using the keys "dimension1", "dimension2", etc... you use "cd1","cd2", etc.

the client var is an instance of "universal-analytics"

**Comment by Bug-Reaper** [comment]
~PR for this should be up in couple of hours.~

PR is up and tested to be reporting sweet sweet custom dimensions with my google analytics events. :)

**Comment by Bug-Reaper** [comment]
I'm actively using a fork of this plugin with proposed PR because I needed it in my production. Anyone else is welcome to do the same by adding this to their package.json:

```
  "dependencies": {
    "analytics": "^0.5.5",
    "@analytics/google-analytics": "github:Bug-Reaper/analytics-plugin-google-analytics-fork"
  }
```

I imagine the PR for this will get merged soon though. Lead dev has quickly fixed other issues I've had with this repo :smile: 

If this won't be fixed anytime soon, somebody should bump the readme to indicate that custom dimension reporting doesn't work if running the google analytics plugin within a node environment. I imagine this is a pretty nice use case, but it's a niche use case that took me a few hours to figure out why it wasn't working and then another 15 hours to fix.

**Comment by DavidWells** [comment]
Thanks a ton for the PR @Bug-Reaper 

You rock! 

This is released with `@analytics/google-analytics@0.5.2` via https://github.com/DavidWells/analytics/pull/87
